### ISSUE TITLE
Error inserting graphics

### ISSUE THREAD

**Issue by aliciaaine** [OpenLiveWriter/OpenLiveWriter#803]
The error had already been corrected when inserting graphics, however, it was raised again.
It shows me again 404 error, bad server.
What's next? Is there a new patch ?, someone to guide me urgently need to be able to post on my blog

**Comment by Shoshiplatypus** [comment]
I am having the same problem. Also, when I insert photos using the workaround (inserting them from Google Photos - very time consuming) I am getting complaints from my blog visitors that they can't see the pictures. I have posted elsewhere on this forum and not had any replies.

If anyone out there is dealing with this forum I should be very, very grateful for some input on how to deal with these ongoing problems. All I am getting at present is a resounding silence.

Answers, please!!

Shoshi

**Comment by CCOSTAN** [comment]
Seems to be Duplicate of issue #786

**Comment by romona** [comment]
When OLW stopped working completely I reluctantly went back to the blogger
platform knowing how much I don't like it. However, when I used it the
first time, I was pleasantly surprized to see that it wasn't the demon it
used to be.

The font I set up in the advanced theme settings for the title, was still
there when I posted. There aren't the same number of options for the text
of the post, but the pictures and text lined up properly. Maybe they've
made a few changes ?

I've had my blog since 2003, so I don't want to take the time to change to
somewhere else. I hope someone/s will be able to bring back OLW from the
graveyard.
